agreement_rebate/models/agreement_type.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2020 Tecnativa - Carlos Dauden
+# Copyright 2020 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import models, fields
+
+
+class AgreementType(models.Model):
+    _inherit = 'agreement.type'
+
+    journal_type = fields.Selection(
```

Comment:
Not sure about this one:

```suggestion
    journal_rebate_type = fields.Selection(
```